src/contents.ts

```diff
@@ -656,4 +675,59 @@ namespace ContentsManager {
      */
     ajaxSettings?: IAjaxSettings;
   }
+
+  /**
+   * Get a path to a file given a relative path.
+   *
+   * @param relativePath - The relative path to the file.
+   *
+   * @param basePath - The optional base path of the file.  The
+   *   default is the server root path.
+   *
+   * #### Notes
+   * If the path is not contained within the server path, it will be
+   * returned unmodified.
+   * If the relativePath starts with a forward slash, the basePath will
```

Comment:
I removed the comment about the forward slash, since that is the default behavior of `resolve` anyway.  I'll try and clean up the language some more.